tests/Operation/FindOneAndUpdateTest.php

```diff
@@ -46,6 +46,10 @@ public function provideInvalidConstructorOptions()
     {
         $options = [];
 
+        foreach ($this->getInvalidArrayValues() as $value) {
+            $options[][] = ['arrayFilters' => $value];
+        }
+
```

Comment:
This can actually go in FindAndModifyTest. Similar to how you added the equivalent test to UpdateTest instead of UpdateOneTest and UpdateMany Test.